batch/Makefile

```diff
@@ -59,12 +65,27 @@ ifneq ($(PY_CHECK),false)
 test-local: $(PY_CHECKERS)
 endif
 test-local: build/conda-env
-	. ../loadconda && conda activate hail-batch && POD_NAMESPACE='test' BATCH_USE_KUBE_CONFIG=1 ./test-locally.sh
```

Comment:
probably should add a line to this makefile:
```
ifndef CLOUD_SQL_PASSWORD
$(error CLOUD_SQL_PASSWORD is not set)
endif
```

Is the password in the usual place?

Reply:
No. I put it in my environment. Not sure what the best strategy is for this.